example/lib/main.dart

```diff
@@ -1,37 +1,36 @@
+import 'package:flame/extensions/vector2.dart';
 import 'package:flame/game.dart';
 import 'package:flame_svg/flame_svg.dart';
-import 'package:flame/position.dart';
 
 import 'package:flutter/material.dart';
 
 void main() {
-  WidgetsFlutterBinding.ensureInitialized();
-
-  final game = MyGame();
-  runApp(game.widget);
+  runApp(GameWidget(game: MyGame()));
 }
 
 class MyGame extends BaseGame {
   Svg svgInstance;
   SvgComponent android;
 
-  MyGame() {
-    _start();
+  @override
+  void render(Canvas canvas) {
+    super.render(canvas);
+    svgInstance?.renderPosition(
+      canvas,
+      Vector2(100, 200),
+      Vector2.all(300),
+    );
   }
+  
+  @override
+  Future<void> onLoad() async {
+    svgInstance = await Svg.load('android.svg');
```

Comment:
Nit: I would use the extension method here

Reply:
Ah, completely forgot about updating the example